lib/bap_core_theory/bap_core_theory_program.ml

```diff
@@ -221,6 +229,14 @@ module Label = struct
       ~public:true
       ~desc:"the program virtual address"
 
+  let bytes = Knowledge.Class.property ~package cls "label-bytes" bytes
```

Comment:
I would suggest `insn-code` as the name of the property, `bytes` is too generic and is more about the representation of datum rather than its contents. But most importantly it is not a property of a label but of an instruction itself. The label is just a generalization of an address, so as a sanity check you can substitute "label" with an "address", e.g., "address-bytes" or "address-code". Is bytes a property of an address? Moreover, is code itself (or a sequence of bytes) is a property of an address? No. It is the property of a machine instruction. So other than changing the name, we also need to move this property to the `Theory.Semantics.cls` class. So, please move it to the `Semantics` module (in the same file).

Reply:
And on the OCaml level, let's name it `code`, e.g., `Theory.Semantics.code`

Reply:
👍